specs/protocol/superchain-config.md

```diff
@@ -29,6 +33,80 @@
 The SuperchainConfig contract is used to manage global configuration values for multiple OP Chains
 within a single Superchain network.
 
+## Configuration Data Structure
+
+The `SuperchainDefinition` type represents the configuration for a
+Superchain target, containing information about L1 contract addresses
+and network parameters.
+
+### SuperchainDefinition
+
+```javascript
+SuperchainDefinition {
+    Name                   string
+    ProtocolVersionsAddr   checksummedAddress
```

Comment:
does checksummed address mean 32 byte array ? not sure what the rest of the spec uses for that, could you please check?